plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1819,6 +1821,98 @@ public void testRegisterTable()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testUnregisterTable()
```

Comment:
Is it intentional to allow unregister managed table?
After following registration table became external and drop does not delete data.

Reply:
It's intentional because the transaction log corruption may happen regardless of managed or external.